src/components/AutomationDetails/index.jsx

```diff
@@ -1,126 +1,181 @@
+          <div className="automation-action">
+            <h1 className="automation-type">{modalContent.type}</h1>
+            <h1 className="dot">.</h1>
+            {slackStatus === 'failure' || emailStatus === 'failure' || freckleStatus === 'failure' ? <h1 className="automation-status">Failed</h1> : <h1 className="automation-status success">Success</h1>}
+            <button type="button" className="retry-button-group">
+              <h1 className="retry">RETRY</h1>
+            </button>
+          </div>
+          <h1 className="automation-date">{formatDates(modalContent.updatedAt)}</h1>
         </div>
       </div>
     );
-  };
-
+  }
 
-  renderModalTable = (modalContent, modalType) => {
-    const { emailAutomations, slackAutomations } = modalContent;
-    const contents = modalType === 'email' ? emailAutomations.emailActivities : slackAutomations.slackActivities;
+  renderChannelTabs = () => {
+    const { modalType } = this.state;
     return (
-      <div className="table-header-details">
-        <table className="details-table">
-          <thead>
-            <tr>
-              {modalType === 'email' ? <th>Sent to</th> : <th>Channel</th>}
-              {modalType === 'email' ? <th>Subject</th> : <th>Auto Type</th>}
-              <th>Status</th>
-            </tr>
-          </thead>
-        </table>
-        <div className="automation-table">
-          {contents.map((content) => {
-            const { emailTo } = content;
-            return (
-              <div key={modalType === 'email' ? emailTo : uuid.v4()} className="table-body-details">
-                <table className="details-table">
-                  <tbody>
-                    <tr>
-                      {this.renderTableData(content, modalType)}
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
-            );
-          })}
+      <div className="channel-tabs-group">
+        <div className="channels-tab-background">
+          <button type="button" className={`automations ${modalType === 'slack' ? 'active' : 'deactivated'}`} id="slack" onClick={this.setCurrentAutomationType}>Slack</button>
+          <button type="button" className={`automations ${modalType === 'email' ? 'active' : 'deactivated'}`} id="email" onClick={this.setCurrentAutomationType}>Email</button>
+          <button type="button" className={`automations ${modalType === 'freckle' ? 'active' : 'deactivated'}`} id="freckle" onClick={this.setCurrentAutomationType}>Freckle</button>
         </div>
       </div>
     );
   }
 
-  renderTableData = (content, modalType) => {
-    const {
-      channelName, type, emailTo, subject, status,
-    } = content;
-    let automationMedia = channelName;
-    let automationTitle = type;
-    if (modalType === 'email') {
-      automationMedia = emailTo;
-      automationTitle = subject;
-    }
+  renderTitles(modalType) {
+    return (
+      <div>
```

Comment:
How about you use [<React.Fragment></React.Fragment>](https://reactjs.org/docs/fragments.html) instead of the empty `<div/>`?

Reply:
Noted!